src/pydata_sphinx_theme/assets/styles/_color.scss

```diff
@@ -0,0 +1,274 @@
+// Colors are defined in rgb string way, "red, green, blue"
+
+/////////////////////////////////////////////////////////////////////////
```

Comment:
We can use the same comment structure as the JS comments:

```scss
/**
  * Some text
  */
```

Reply:
None of the existing files are using this structure. my idea was to have some section header that standout in file overview like VS code. Instead of the suggested strucure, I used a simple `// === section header ===` as suggested in this article: https://perishablepress.com/obsessive-css-code-formatting-organization-comments-and-signatures/